nix/pkgs/wire-binaries.nix

```diff
@@ -8,14 +8,14 @@ let
   # These values are manually kept in sync with:
   # https://github.com/kubernetes-sigs/kubespray/blob/release-2.20/roles/download/defaults/main.yml
```

Comment:
You seem to have updated `kubespray`. Is this comment still fine. Or, does it need to be updated, too?

Reply:
needs to be updated..fixed :)